cypress/integration/ete/registration_3.8.cy.ts

```diff
@@ -1235,7 +618,7 @@ describe('Registration flow - Split 2/2', () => {
 					timeRequestWasMadeInitialEmail,
 				);
 
-				cy.interceptRecaptcha();
+				cy.interceptRecaptcha(3);
```

Comment:
Here is where we increase the number of times we intercept recaptcha requests from 1 to 3 ... this is so that we also catch any retry attempts if they happen.